cfme/tests/infrastructure/test_infra_tag_filters_combination.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from cfme import test_requirements
+from cfme.infrastructure.cluster import Cluster
+from cfme.infrastructure.provider import InfraProvider
+from cfme.infrastructure.virtual_machines import Vm, Template
+from fixtures.provider import setup_one_or_skip
+from utils.providers import ProviderFilter
+from utils.update import update
+
+
+pytestmark = [test_requirements.tag, pytest.mark.tier(2)]
+
+test_items = [
+    ('clusters', Cluster),
+    ('vms', Vm),
+    ('templates', Template)
+]
+
+
+@pytest.fixture(scope='module')
+def a_provider(request):
+    prov_filter = ProviderFilter(classes=[InfraProvider],
+                                 required_fields=['datacenters', 'clusters'])
+    return setup_one_or_skip(request, filters=[prov_filter])
+
+
+@pytest.fixture(params=test_items, ids=[str(test_id[0]) for test_id in test_items],
```

Comment:
OPTIONAL: Why is ``str`` necessary? I would use ``collection_name`` here for consistency instead of ``test_id`` and tuple unpacking for readability:
``[collection_name for collection_name, __ in test_items]``